packages/Input/Input.md

```diff
@@ -14,12 +14,20 @@ For information on how to use disabled Inputs, please refer to the [disabled for
 ```jsx
 <Box between={2}>
   <Input label="First name" value="Harry" />
-
   <Input type="tel" label="Mobile phone" pattern="[0-9]{10}" maxLength="10" />
   <Input type="password" label="Tax ID" value="123456789" autoComplete="on" />
 </Box>
 ```
 
+### Usage criteria
```

Comment:
Let's have usage criteria under every major feature and please add the following regarding hint text:
Hint can now be used inline (beside) and below the label

Reply:
What does "under every major feature" mean?

So the points should be updated to this?
> - Hint can now be used inline (beside) and below the label
> - Use `hintPosition` to re-position the `hint` below the `label` if the inline `hint` wraps
>   - The `hint` should **not** be longer than 85 characters
>   - The `hint` should **not** wrap more than 3 lines on all viewport sizes

Reply:
Maybe they're not major features? But I meant these ones under forms:
Button, Input, Checkbox, Radio, ButtonGroup, Select, TextArea, InputFeedback, Tooltip

Reply:
Should another ticket be used to discuss the scope of moving _Usage Criteria_ for all or some components to the top of each `readme.md` ?